frontend/src/components/Form.js

```diff
@@ -41,11 +41,45 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
+/**
+ * Validates from inputs on submission and alerts user on error
+ * @param {String} role
+ * @param {String} productArea
+ * @param {Number} duration
```

Comment:
document the time unit here. From the conversion below it looks like minutes?

Reply:
Added!